python/cudf_polars/cudf_polars/dsl/utils/windows.py

```diff
@@ -110,6 +110,10 @@ def duration_to_scalar(dtype: plc.DataType, value: int) -> plc.Scalar:
         return plc.Scalar.from_py(
             value // 1_000_000, plc.DataType(plc.TypeId.DURATION_MILLISECONDS)
         )
+    elif tid == plc.TypeId.TIMESTAMP_DAYS:
```

Comment:
This change is unrelated to the PR. I decided to make a separate PR for it: #19689